### DISCUSSION TITLE
Setting up - no data logging

### DISCUSSION THREAD

**Discussion by widget567** [artisan-roaster-scope/artisan#752] [Q&A]
Hi guys,

Currently trying to set up artisan to my Kuban 5kg Base 5 (with usb data bridge to phidget 1048_0)  to Mac OS  12.0.1 and having some difficulty. Have followed the setup instructions, but am getting no data logging on artisan. Have also tried on a windows laptop and having similar issues.

After downloading the phidget control panel I can see that it is connected and receiving some readings. However, nothing appears to be showing on my network settings.

Also using a hotspot as I don't currently have any wifi in the roastery.

Any help would be greatly appreciated!

![image (3)](https://user-images.githubusercontent.com/95690700/145032233-cd3de85f-8239-4e92-8a05-aaf2ca958c2e.png)

![i
![image (2)](https://user-images.githubusercontent.com/95690700/145032077-e336118f-ef6a-4011-a0bd-0669e7ff324f.png)
mage](https://user-images.githubusercontent.com/95690700/145031965-f80a50a0-b9da-41f4-9352-1945d7d0f107.png)
![image (4)](https://user-images.githubusercontent.com/95690700/145032385-d4bf6af3-289b-4263-842d-78130ea3aeb1.png)

Thanks a lot,

Ben

**Comment by CarefreeBuzzBuzz** [comment]
Did you install the Phidgets Driver (not just the control panel) the full driver farther down on the Phidgets page.  
----------------------------------------
Michael Herbert, Head Roaster
Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com/>
Artisan RoasterScope  <https://artisan-scope.org/>Team

**Reply by widget567** [reply]
Thanks for the quick response. I believe so, I installed this from the Phidgets website: 
![image](https://user-images.githubusercontent.com/95690700/145035725-244daa20-ca7d-41b4-bb3b-1bbf5a3eaad7.png)

**Comment by CarefreeBuzzBuzz** [comment]
Then if you are getting readings in the Control panel ( you didn’t expand it to show them), you need to tell us how you set up Artisan with a GREAT DEAL of specifics.  There are Machine installations for typical Phidgets you can run in Artisan from the menu.  Did you try any of these?  Remember no one knows your roaster or what the devices are or how they connect.  So please give all the details.  Did your mfg give any instructions?  
----------------------------------------
Michael Herbert, Head Roaster
Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com/>
Artisan RoasterScope  <https://artisan-scope.org/>Team

**Reply by widget567** [reply]
Ok my apologies, I'll try and give you as much detail as I can now! 

So far we have followed the setup guide for our machine type: https://artisan-scope.org/machines/kuban/ - and created a static IP address and inputted the subnet mask. 

We then installed the Phidgets driver and interface as per the instructions on their website. 

We then followed some instructions sent by our manufacturer: 
- On artisan, we selected config -> machine -> Phidget -> USB 1048 Databridge 2xTC 
- On artisan we selected config -> device -> extra devices , and added 2  devices as shown in the screenshot (device name: Phidget  1048 4xTC 01)
- We changed the 'phidgets' settings under device assignment to the settings shown in the second screenshot

We then attempted to take readings from the roaster, but no changes were registered. No errors are found in the error log  either. 

Hopefully this makes some more sense. This is very new to us so apologies if I'm missing something obvious!

Many thanks,

Ben

![image](https://user-images.githubusercontent.com/95690700/145047654-ed7c0433-6b51-496a-addd-79204a078670.png)

![image](https://user-images.githubusercontent.com/95690700/145048297-5fa9e848-9178-46b5-8739-62e4b50b7620.png)

**Comment by CarefreeBuzzBuzz** [comment]
I am guessing then its a network set up issue. Are you seeing readings for any of the devices in the Phidgets control panel?  I am not familiar with this type of network setup so not sure I can help much.  


Are you sure you did all these steps correctly -  Your computer is a different IP address than the roaster - did you confirm what the roaster default is?  Both devices on the same network. 
Permalink <https://artisan-scope.org/machines/kuban/#setup>
The computer running Artisan must be on the same IP network as the roaster. The default IP address of the roaster is 192.168.0.5. Configure your computer to use a static IP address in the range of the roaster (192.168.0.x) but with x different from that of the roaster (eg. 192.168.0.42). Choose 255.255.255.0 as subnet mask.

This can be done on Windows using the Network Sharing Center by adding a TCP/IPv4 Local Area Connection with those properties. On macOS you set your ethernet port in the Network Control panel to “IPv4: Manually” and fill in the IP and subnet mask accordingly.

----------------------------------------
Michael Herbert, Head Roaster
Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com/>
Artisan RoasterScope  <https://artisan-scope.org/>Team

**Comment by CarefreeBuzzBuzz** [comment]
And just to be clear your phidgets are connected to the roaster and the roaster to the computer via the network?  The Phidgets aren’t connected to the MAC directly.  

Also you CANT run the Phidgets panel open while trying to use Artisan.
----------------------------------------
Michael Herbert, Head Roaster
Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com/>
Artisan RoasterScope  <https://artisan-scope.org/>Team

**Reply by widget567** [reply]
Ahh ok, I had the Phidgets panel open, oops! Thanks a lot. The readings are now appearing, although they are 30 degrees celcius or so above the roaster reading. Any idea why this would be?

**Comment by CarefreeBuzzBuzz** [comment]
Roaster isn’t reading the same probes is my guess.  Very common issue.  Different probes, different devices, different placement.  I woundnt spend time on it.  I don’t use my roaster readings except for safety cut off.  


----------------------------------------
Michael Herbert, Head Roaster
Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com/>
Artisan RoasterScope  <https://artisan-scope.org/>Team
